### DISCUSSION TITLE
There is trace information in skywalking, but there is no information such as p95

### DISCUSSION THREAD

**Discussion by ouyangli** [apache/skywalking#8584] [Q&A] [answered]
question:
1. The endpoint /gateway/createConnection has trace information, but no information such as p95
2. Does skwalking support p95 for asynchronous operations?
![image](https://user-images.githubusercontent.com/8622409/154782772-d297e7be-7e2b-4d4b-8ae8-96a9e90afe39.png)

![image](https://user-images.githubusercontent.com/8622409/154782679-26604750-04bb-499f-bbce-450bd33b742b.png)

**Comment by wu-sheng** [comment, marked as answer]
This is a span, not must be an endpoint. Only the operation name of entry spans or logic spans could be considered as endpoints. 

**Endpoint. A path in a service for incoming requests, such as an HTTP URI path or a gRPC service class + method signature.**

 Quoted from https://skywalking.apache.org/docs/main/v8.9.1/en/concepts-and-designs/overview/#why-use-skywalking
